eng/pipelines/common/templates/wasm-debugger-tests-firefox.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
```

Comment:
1. Instead of a new template, add a `browser` parameter to `wasm-debugger-tests.yml`, with a default value of `chrome`
2. In that, pass `/p:RunInFirefox=1` if the `browser == firefox`
3. And use that for adding the new Firefox jobs

Reply:
Something like https://github.com/radical/runtime/commit/655c0f17241c571fd68f0983a1c492be7a575ecf

Reply:
done.